tasks/VisitMeasureTasks.py

```diff
@@ -0,0 +1,6 @@
+import astropy.units as u
+from lsst.pipe.base import Struct, Task
+from lsst.pex.config import Config
+from lsst.verify import Measurement
+
+#Currently this is a placeholder until additional per-visit measurements are added.
```

Comment:
I'm a little uncomfortable having a file with no executable code in it, but since we will presumably have per visit metrics relatively soon, I'm not going to argue too hard against it.

Reply:
Right, I had the same thought. Maybe we can add a simple per-visit metric this week.